using-mobile-extensions/adobe-edge-consent/README.md

```diff
@@ -0,0 +1,165 @@
+# Adobe Experience Platform Edge Consent
+
+## Configure the Adobe Experience Platform Edge Consent extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Consent Collection** extension, and click **Install**.
+3. Set your desired default consent level.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+### Configure the AEP Edge Consent extension
+
+![Adobe Edge Network extension configuration](/Users/nporter/Developer/Documentation/acp-sdks-documentation/.gitbook/assets/mobile-edge-consent-launch-configuration.png)
+
+
+
+## Add the AEP Edge Consent extension to your app
+
+### Download and import the Edge Consent extension
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+
+   ```java
+   implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+   implementation 'com.adobe.marketing.mobile:edge:1.+'
+     implementation 'com.adobe.marketing.mobile.edgeidentity:1.+'
+   implementation 'com.adobe.marketing.mobile.edgeconsent:1.+'
+   ```
+
+2. Import the Mobile Core and Edge extensions in your application class.
+
+   ```java
+    import com.adobe.marketing.mobile.*;
+   ```
+
+   {% endtab %}
+
+{% tab title="iOS" %}
+
+1. Add the Mobile Core and Edge extensions to your project using Cocoapods. Add following pods in your `Podfile`:
+
+   ```swift
+   use_frameworks!
+   target 'YourTargetApp' do
+       pod 'AEPCore'
+       pod 'AEPLifecycle'
+       pod 'AEPSignal'
+       pod 'AEPEdge'
+       pod 'AEPEdgeIdentity'
+       pod 'AEPEdgeConsent'
+   end
+   ```
+
+2. Import the Mobile Core and Edge libraries:
+
+### Swift
+
+```swift
+// AppDelegate.swift
+import AEPCore
+import AEPEdge
+import AEPEdgeIdentity
+import AEPEdgeConsent
+import AEPLifecycle
+import AEPSignal
+```
+
+### Objective-C
+
+```text
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+@import AEPEdgeConsent;
+@import AEPLifecycle;
+@import AEPSignal;
+```
+
+{% endtab %}
+{% endtabs %}
+
+### Register Edge with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+      MobileCore.configureWithAppID("yourAppId");
+
+      try {
+        Edge.registerExtension();
+        Consent.registerExtension(); // register Consent
+        com.adobe.marketing.mobile.edge.identity.Identity.registerExtension();
```

Comment:
If you change the dependencies to just include core and not sdk-core, then this can just be `Identity.registerExtension()`.